README.md

```diff
@@ -137,26 +180,22 @@ Next to the save button is a button that will allow you to export your results.
 
 - CSV - All the columns in the table will become columns in the CSV
 - Markdown - The columns will become frontmatter data and the children of the block or page will become markdown content.
-
-### Query Blocks
-
-The above component is also available as a block component. This allows you to create several on a page, wherever on the page you want. To create one, simply add `{{query block}}` to any block on the page.
-
-### Query Drawer
-
-The above component is also available as a left hand drawer, accessible from the command palette. This allows you to execute a query no matter where in your graph you are. To open, enter `Open Query Drawer` from the Roam Command Palette.
-
 ### Styling
 
-Every Query Page is rooted with a `div` that has an `id` of `roamjs-query-page-${uid}` where `uid` is the block refence of the query block or the page reference of the page. You could use this id to style individual queries with affecting other ones.
+Every [Query Page](#query-pages) or [Query Block](#query-blocks) is rooted with a `div` that has an `id` of `roamjs-query-page-${uid}` where `uid` is the block reference of the query block or the page reference of the page. You could use this id to style individual queries with affecting other ones.
 
 ### SmartBlocks Integration
 
-This extension comes with its own SmartBlocks command! The `<%QUERYBUILDER%>` command will run an existing query builder instance in your graph and return the results as separate blocks. The command takes in two arguments:
+This extension comes with its own SmartBlocks command! The `<%QUERYBUILDER%>` command will run an existing [Query Block](#query-blocks) or [Query Page](#query-pages) instance in your graph and return the results as separate blocks. The command takes in two arguments:
 
-1. The block reference of the query builder instance
+1. The block reference of the query instance
 1. The format to output each result in. You can use placeholders, like `{text}` to insert the value from the result in. There's a placeholder available for each Selection label used in the query.
 
+> The block reference of a `Query Page` can be found at the end of the `URL`
```

Comment:
Ah, but there is no bullet for `Query Pages` either.  Or do you see one?
Maybe we need to add a menu item "Copy Block Ref"?  But it might be a lot of clutter for a small use case.

![image](https://user-images.githubusercontent.com/3792666/214720490-6d470c71-df1c-4a54-b3d4-8976759bac51.png)

For now, app users can still get the link this way, then paste it somewhere to get the block ref.  Definitely not ideal.

![image](https://user-images.githubusercontent.com/3792666/214720642-36698f5c-c3fc-43f8-bfaf-86a11b7cefa1.png)

Reply:
right, makes sense. This is probably the best way for now